notebooks/niriss_ami_binary/2_niriss_ami_binary.ipynb

```diff
@@ -11,19 +11,21 @@
    "cell_type": "markdown",
```

Comment:
<p>A couple more comments on this cell:</p><p> </p><ol><li>I get a new error here about a missing rateints file that I think is related to the use of <code>Path</code> objects. Converting the <code>df_</code> variables to strings in the three places they're used with <code>run()</code> (e.g., <code>result1.run(str(df))</code>) fixes it for me.</li><li>On the line that defines <code>df_rate</code>, I think you accidentally put <code>'rateints'</code> in the call to <code>replace()</code> instead of <code>'rate'</code>.</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jdat_notebooks/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Good catch! Thank you!